javascript/src/main/components/Nav/AuthNav.js

```diff
@@ -1,11 +1,24 @@
 import React from "react";
 import { useAuth0 } from "@auth0/auth0-react";
+import { Navbar } from "react-bootstrap";
 import LoginButton from "./LoginButton";
 import LogoutButton from "./LogoutButton";
 
 const AuthNav = () => {
-  const { isAuthenticated } = useAuth0();
-  return <>{isAuthenticated ? <LogoutButton /> : <LoginButton />}</>;
+  const { user } = useAuth0();
+  if(user) {
+    const { name, picture, email } = user;
+    return <><Navbar.Text style={{marginRight: 15}}>{"Hello, " + name}
+      </Navbar.Text>      <img
+            src={picture}
+            alt="Profile"
+            className="rounded-circle"
+            width="36"
+            style={{marginRight: 15}}
+      /><LogoutButton /></>
+  } else {
+    return <><LoginButton/></>
+  }
 };
```

Comment:
I'd suggest a slightly different formatting to make things clearer.

```suggestion
  if(user) {
    const { name, picture, email } = user;
    return <>
      <Navbar.Text style={{marginRight: 15}}>{"Hello, " + name}</Navbar.Text>  
      <img
            src={picture}
            alt="Profile"
            className="rounded-circle"
            width="36"
            style={{marginRight: 15}}
      />
      <LogoutButton />
    </>
  } else {
    return <LoginButton/>
  }
};
```